kubemarine/resources/schemas/definitions/services/loadbalancer.json

```diff
@@ -94,6 +94,30 @@
           "description": "Target https port"
         }
       }
+    },
+    "keepalived": {
+      "type": "object",
+      "description": "The section contains the configuration parameters that are applied to the keepalived.conf config file",
+      "properties": {
+        "global": {
+          "type": "object",
+          "description": "Parameters that are passed directly to the 'global_defs ' section of keepalived.conf file.",
+          "properties": {
+            "vrrp_garp_master_refresh": {
+              "type": "integer",
+              "description": "minimum time interval (in seconds) for refreshing gratuitous ARPs while MASTER."
+            }
+          }
+        },
+        "config": {
```

Comment:
I am trying to understand use case of this property and concluded that it may be used only for add_node procedure. In this case, it is possible to have custom configuration for each balancer. Do you have other ideas?

In such a case, the inventory will not describe the cluster, as the inventory can describe only one balancer. I doubt that it is worth adding such contradictory feature, and anyway suggest to describe this the only use case in the doc.

Reply:
Yes, the most suitable case for following option is one-by-one balancer addition. In that case user install cluster with one balancer only, then fix keepalived config and run add_node procedure for every other balancer.
I think, we do not need to fully support different configuration for different balancers, because it's to rare case and requires a lot of inventory/code changes.
Added more explicit description for that parameter.